collector.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/ffddorf/unms-api-go"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type unmsCollector struct {
+	client    *unms.APIClient
+	clientCtx context.Context
+	site      string
+	up        *prometheus.Desc
+}
+
+const namespace = "unms"
+
+func NewUnmsCollector(client *unms.APIClient, clientCtx context.Context, site string) prometheus.Collector {
+	c := unmsCollector{
+		client:    client,
+		clientCtx: clientCtx,
+		site:      site,
+		up: prometheus.NewDesc(
+			namespace+"_"+"device_up",
+			"If device is connected to UNMS.",
+			[]string{"name", "id"},
```

Comment:
"name" is not stable, use "id" and implement a null metric that relates those together

Reply:
add "site_id" label